pandasql/core.py

```diff
@@ -432,6 +432,10 @@ def __init__(self, data=None, name=None, sources=None, deep_copy=False,
             # Store columns
             self.columns = df.columns
 
+        elif loaded_on_sqlite is True:
+            self.columns = df.columns
```

Comment:
Why are we using `df` here? I would think that when we're loading directly into SQLite, we don't keep a copy on Pandas, and so, the `data` argument to this function should be `None`.

Reply:
`df` here (in the case of a direct load to SQLite) refers to the empty dataframe used to send the column names. This way we have them for projections, group_bys, etc.

Reply:
Yep, but `_cached_result` gets set to it right? That's what will break things because PandaSQL will think it has a cached result on Pandas, but it actually doesn't. Lmk if I'm missing something

Reply:
nvm I understand what you mean